ui/src/styles/device.ts

```diff
@@ -2,6 +2,7 @@ const size = Object.freeze({
   mobileS: "320px",
   mobileM: "375px",
   mobileL: "425px",
+  mobile: "1024px",
```

Comment:
Why is mobile === laptop?

Reply:
I didn't want to break the code that Hanlin and Jeffrey were working on since they both used laptop. The correct size for the mobile is 1024px, I'm not sure why laptop was originally set to that.

Reply:
Open up a ticket after this gets merged to standardize it?